index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Dos App</title>
+</head>
+  <body>
+      <h1>To-Dos</h1>
+      <form action="#">
+        <input type="text" id="taskInput" placeholder="Enter a task">
+        <input type="submit" value="+">
+      </form>
+      <br>
+      <form id="checkboxForm">
+        <!-- Checkboxes will be added here -->
+        <button type="submit">Submit Response</button>
+      </form>
+      <menu>
+        <li><button id="allTasksBtn">All Tasks</button></li>
+        <li><button id="completeTasksBtn">Complete Tasks</button></li>
+        <li><button id="incompleteTasksBtn">Incomplete Tasks</button></li>
+      </menu>
+  </body>
+</html>
```

Comment:
@jjw18th Please configure your code editor to always add a blank space at the end of your files. Also look into why this is a standard practice everywhere and the benefits it brings.